c/include/libsbp/observation.h

```diff
@@ -612,6 +612,21 @@ coordinate system
 } msg_almanac_glo_t;
 
 
+/** GLONASS SV orbital and frequency slots mapping information
+ *
+ * The message reports mapping information regrding GLONASS SV orbital and
+ * frequency slots. Frequency numbers -7 to 6 have been reported as 1 to 14
```

Comment:
Let's make the description a bit differently. Maybe providing a table like this:
```
index (SV orbital slot)  fcns[index]
0                        0xFF
1                        FCN for SV orbital slot 1
...                      ...
28                       FCN for SV orbital slot 28
29                       0xFF
30                       0xFF
31                       0xFF
```
BTW, why 32 slots in `fcns`?